### PR TITLE
Lightning fighters

### PR DESCRIPTION
**Pull request by silverhook**
Mostly handles https://github.com/Alshie/does-it-flip/issues/11

But there are other games mentioned in that ticket, which need addressing as well.

### CODE REVIEW

File: games.json [hunk 1]
```diff
@@ -889,5 +889,17 @@
     "categories": "Shooter, Action, Arcade",
     "players": "1",
     "publisher": "Chorus Worldwide Games"
+  },
+  {
+    "name": "LIGHTNING FIGHTERS",
+    "collection": "Arcade Archives",
+    "additional_info": "This game is also known under the name Trigon.",
```
Comment (silverhook) [inline games.json:896]: @riccyjay, does this make sense?

Comment (riccyjay) [reply games.json:896]: Everything except the publisher, who should be Hamster, I think.

Comment (silverhook) [reply games.json:896]: It is Hamster (see line 902 below). Or do you mean the caps?

**Comment by silverhook** [comment]
fixes #11

File: games.json (hunk 1 above)
Comment (riccyjay) [reply games.json:896]: I see. The email I got looked like it said Chorus Worldwide - but I might be confusing things (I don't know coding at all lol)

Comment (silverhook) [reply games.json:896]: That bit belongs to the game above it. 
For changes see this: https://github.com/Alshie/does-it-flip/pull/14/files

Yeah, GitHub can be a bit confusing for non-coders (I should know, haha).

**Review by Alshie — APPROVED** [review]

**Pull request merged**